### ISSUE TITLE
Core certificate not being created within startup

### ISSUE THREAD

**Issue by gabrielpulga-grafeno** [kubernetes/kubernetes#111938]
### What happened?

Been having issues on this new cluster I've built with k8s

pod1
`MountVolume.SetUp failed for volume "kube-api-access-2qx96" : object "app"/"kube-root-ca.crt" not registered
`
pod2
```
MountVolume.SetUp failed for volume "kube-api-access-8x9l6" : object "app"/"kube-root-ca.crt" not registered
What does this mean? What is this certificate? Why this is happening?
```

I'm confused if this is something related to RBAC and how do I register this kube-root-ca.crt

### What did you expect to happen?

This certificate should already be in there

### How can we reproduce it (as minimally and precisely as possible)?

kubectl apply -k .
see events of pods

### Anything else we need to know?

this is running in eks 1.22

### Kubernetes version

<details>

```
1.22
```

</details>

### Cloud provider

<details>
aws
</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

### Container runtime (CRI) and version (if applicable)

<details>

</details>

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by liggitt** [comment]
What point release of 1.22 are you running? Known examples of this issue were fixed in 1.22.9+ (see https://github.com/kubernetes/kubernetes/issues/105204#issuecomment-1104427967)

**Comment by dims** [comment]
please re-open with additional information
